contracts/mixnet/src/queries.rs

```diff
@@ -93,6 +93,10 @@ pub(crate) fn query_layer_distribution(deps: Deps) -> LayerDistribution {
     read_layer_distribution(deps.storage)
 }
 
+pub(crate) fn query_total_mix_stake(deps: Deps) -> Uint128 {
+    total_mix_stake(deps.storage).load().unwrap()
```

Comment:
I'd write a comment here about why the unwrap here is fine - initially it scared me until I looked at the call stack.